### ISSUE TITLE
Do we want the archive data to be replaced with V3 data? 

### ISSUE THREAD

**Issue by mccabete** [Earth-Information-System/fireatlas#139]
Right now the API has the archive data from V2, and NRT from V3. Do we want all API data to be V3?

**Comment by mccabete** [comment]
Personally, I think yes! @eorland would take affect your paper release? We could store the V2 data in a different public repo somewhere.

**Comment by eorland** [comment]
Yeah good q! I'm all in favor of switching over to v3. I plan on also doing a v2 release with the paper, so maybe that can be where it lives. I don't see any issue.

**Comment by zebbecker** [comment]
Consolidating discussion about data sharing to #184 from across several issues
